site/pages/react/how-to-set-up-smart-account-client.mdx

```diff
@@ -0,0 +1,53 @@
+| `policyId`          |                                        `string` |                                                                                                                                                                                                       gas manager policy ID from the [Dashboard](https://dashboard.alchemy.com/gas-manager) |
+| `useSimulation`     |                                       `boolean` |                                                                                                                                                                      enables use of [`alchemyUserOperationSimulator`](/reference/account-kit/infra/functions/alchemyUserOperationSimulator) |
+| `customMiddleware`  |                            `ClientMiddlewareFn` |                                                                                                                                                                                                                                      custom middleware to run before requesting sponsorship |
+| `feeEstimator`      |                            `ClientMiddlewareFn` |                                                                                 function that estimates the fees for a user operation (if you are using our RPCs, it's important to use the [`alchemyFeeEstimator`](/reference/account-kit/infra/functions/alchemyFeeEstimator) middleware) |
+| `gasEstimator`      |
+| `signUserOperation` |                            `ClientMiddlewareFn` |                                                                                                                                                                                                                            function that sends a user operation or catch of user operations |
+| `opts`              | ` z.input<typeof SmartAccountClientOptsSchema>` |                                                                                                                                                                                                                                                                             further options |
+| `accountParams`     |                                 `AccountConfig` | object with the following fields <ul><li>`mode` : "default" \| "7702"</li><li>`entryPoint`: of type `EntryPointDef`</li><li>`signerEntity`: of type `SignerEntity`</li><li>`salt`: of type `bigint`</li><li>`factoryAddress`: of type `Address`</li><li>`initCode`: of type `Hex`</li></ul> |
+
+:::tip
+We currently do not support using a third-party signer at the React hook level. To do so you must use the `createSMAV2Client` method
+:::
+
+## Legacy Users: other account options
+
+The following are the other smart accounts you can use
+
+|                    | LightAccount | MultiOwnerLightAccount | MultiOwnerModularAccount |
```

Comment:
i wonder if it might be nice to show MAv2 here too just for comparison that it can do everything?

Reply:
great idea!